keras_nlp/benchmarks/README.md

```diff
@@ -25,3 +25,28 @@ the following results were obtained:
 To change the configuration, say, for example, number of layers in the transformer
 model used for inference, the user can modify the config dictionaries given at
 the top of the script.
+
+
+## Sentiment Analysis
+
+
+For benchmarking classification models, the following command can be run
+from the root of the repository:
+
+```sh
+python3 .keras_nlp/benchmarks/sentiment_analysis.py \
+    --model="bert" \
+    --preset="bert_small_en_uncased" \
+    --learning_rate=5e-5 \
+    --num_epochs=5 \
+    --batch_size=32
+```
+
+flag `--model` specifies the model name, and `--preset` specifies the preset under testing. `--preset` could be None, 
+while `--model` is required. Other flags are common training flags.
+
+This script outputs:
+
+- validation accuracy for each epoch.
+- testing accuracy after training is done.
+- total elapsed time (in seconds).
```

Comment:
Will this display in some way the average train step time during fit? I think that is an important metric for us.

Reply:
I think I can do that by creating a Callback subclass. Do you have a more straightforward suggestion?

Reply:
I'm not sure actually! Let me get back to you.

I know we would like to have something indicative of the time elapsed for only the forward and backwards pass. The total elapsed time will also include compilation (which is good too, but not always the metrics we will want).

Reply:
Actually, how about this? Btw After the "Compiler cluster using XLA" line, I had to wait more than 10 mins. I don't know the reason maybe it is because of WSL2

![image](https://user-images.githubusercontent.com/19870022/213012004-c9669529-7385-45a3-8709-c9862c173cfb.png)

Reply:
Yeah, I think the train step time printed by model fit is totally fine at least for now.

Maybe as a follow up we should add a flag to enable tensorboard and the tensorboard profiler. https://www.tensorflow.org/tensorboard/tensorboard_profiling_keras